docs/platform/howto/byoc/networking-security.md

```diff
@@ -0,0 +1,124 @@
+| ---------------------- | ---------------| --------------------------------------------------------------------------------------------------------------------------------------------- |
+| Aiven Management plane | Bastion subnet | Aiven uses SSH from the Aiven Management plane to the bastion service node for troubleshooting, setup, or configuration tasks on the bastion. |
+| Aiven Management plane | Bastion subnet | Proxy service is used to tunnel the traffic from the Aiven Management plane to workload nodes.                                                |
+| Aiven Management plane | Bastion subnet | Aiven Management plane accesses the bastion service node to retrieve status information and perform routine operations.                       |
+
+### Traffic from the bastion
+
+| Source         | Destination      | Description                                                                                                                             |
+| -------------- | -----------------| --------------------------------------------------------------------------------------------------------------------------------------- |
+| Bastion subnet | Aiven Management | Communication channel with the Aiven Management plane used to collect metrics and logs from the bastion and workload nodes              |
+| Bastion subnet | Aiven Management | Required for the bastion and workload nodes to make calls to the Aiven Management plane                                                 |
+| Bastion subnet | AWS CloudFront   | Communication channel for downloading RPM packages for bastion and workload functionality and maintenance                               |
+| Bastion subnet | Workload subnet  | Aiven uses SSH from the bastion subnet to the workload subnet for troubleshooting, setup, or configuration tasks on the workload nodes. |
+| Bastion subnet | Workload subnet  | Aiven Management plane accesses the workload nodes via the bastion node to retrieve status information and perform routine operations.  |
+| Bastion subnet | DNS & NTP        | Destination dependant on cloud provider                                                                                                 |
+
+:::note
+Aiven Management IPs are considered relatively stable but do occasionally change and are
+considered dynamic for the purposes of firewall policies. To that end, an egress of
+`0.0.0.0/0` is generally required to accommodate traffic to Aiven Management.
+:::
+
+## Workload nodes networking
+
+Management traffic to and from backend workload nodes is sent through the bastion proxy and
+encrypted. Aiven Management and API calls are always encrypted. SSH access is permitted if
+Aiven support staff require direct access to the bastion service or workload nodes. SSH
+connections from support staff are logged, monitored, and require validation by Aiven’s
+security operations team. SSH connections originate from the redundant Aiven gateways.
```

Comment:
```suggestion
security operations team. SSH connections originate from Aiven Management plane's NAT gateways.
```

Reply:
Do we really need to say that Aiven's gateways are NAT gateways? I even think they are some in-house `crabredir` proxies.

Reply:
Aiven's gateways (`gw1` and `gw2`) are not NAT gateways by the commonly understood meaning of that term. You could argue that they tunnel connections through proxy programs to 'translate' the source address of the connections somehow, but that'd just confuse the majority of the audience who would understand NAT as packet-level IPv4 Network Address Translation.